### PR TITLE
Fix getAmountsIn and getAmountsOut to not show ROOT symbol

### PR DESCRIPTION
**Pull request by KarishmaBothara**
## Summary

Fix the symbol on Dex rpc call

image

after this PR

image

## Checklist

- [x] Add description
- [ ] Tag related issue(s)
- [ ] Add appropriate tests

### CODE REVIEW

**Review by ken-centrality — DISMISSED** [review]
LGTM, do we need to update this in our `getApiOptions` in `trn-js-api`? also, maybe worth updating it there and import it here?

**Comment by KarishmaBothara** [comment]
> LGTM, do we need to update this in our `getApiOptions` in `trn-js-api`? also, maybe worth updating it there and import it here?

sure.. will update it in trn-js-api.. and once we have a new release..will use it here

**Comment by KarishmaBothara** [comment]
Updated https://github.com/futureversecom/trn-js-api/pull/14
Once we have a new release, will update this PR

**Comment by ken-centrality** [comment]
@KarishmaBothara trn-js-api is update with types, you can use them here now

**Comment by KarishmaBothara** [comment]
Using the new @therootnetwork/* packages..  have one issue to render RPC calls.. looking at it..
Screenshot 2023-08-29 at 1 07 40 PM

**Comment by KarishmaBothara** [comment]
> Using the new @therootnetwork/* packages.. have one issue to render RPC calls.. looking at it..
> Screenshot 2023-08-29 at 1 07 40 PM

Solved this via typeBundle..
However, by adding new repo, it has introduced lot of lint error.. Have ignored it for now, as no changes were made to those file.. But we might want to rethink on this

**Comment by ken-centrality** [comment]
`@therootnetwork/api@1.0.5` is up

**Comment by KarishmaBothara** [comment]
> `@therootnetwork/api@1.0.5` is up

works with version

File: packages/react-components/src/InputRpc/options/method.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import React from 'react';
 import { ApiPromise } from '@polkadot/api';
 
 export default function createOptions (api: ApiPromise, rpcs: Record<string, Record<string, DefinitionRpcExt>>, sectionName: string): DropdownOptions {
+  console.log('rpcs::', rpcs);
```
Comment (ken-centrality) [inline packages/react-components/src/InputRpc/options/method.tsx:12]: remove logging

File: packages/react-components/src/InputRpc/rpcs.ts
Comment (ken-centrality) [inline packages/react-components/src/InputRpc/rpcs.ts:1]: remove commented out code if we don't need it

**Review by ken-centrality — COMMENTED** [review]
Looks good to me, just a few nits...

Btw, is there anyway we can actually resolve the eslint issue rather than sprinkling the ignore comments everywhere? This might cause some conflict when we want to rebase with polkadot.

**Pull request merged**